### PR TITLE
[DS-4507] nginx regrite php extensions

### PR DESCRIPTION
**Pull request by mcappato**
Related to [DS-4507](https://makingsense.atlassian.net/browse/DS-4507)

[DS-4507]: https://makingsense.atlassian.net/browse/DS-4507?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MatiasRoy-Rodriguez — APPROVED** [review]
Good!

File: index.php [hunk 1]
```diff
@@ -26,13 +26,13 @@
...
             <div class="emms__header__logo">
-                <a href="./index.php"><img src="src/img/logos/logo-emms.png" alt="Emms 2023"></a>
+                <a href="/"><img src="src/img/logos/logo-emms.png" alt="Emms 2023"></a>
             </div>
             <a class="emms__header__nav--mb" id="btn-burger"></a>
             <nav class="emms__header__nav emms__header__nav--hidden" id="nav-mb">
                 <ul class="emms__header__nav__menu">
                     <li><a href="#" class="active">home</a></li>
-                    <li><a href="./ecommerce.php">e-commerce</a></li>
+                    <li><a href="./ecommerce">ecommerce</a></li>
```
Comment (mcappato) [inline index.php:35]: @MatiasRoy-Rodriguez @ornella-688  me confirman cual es el content correcto aqui?

File: index.php [hunk 2]
```diff
@@ -253,7 +253,7 @@
                     </ul>
                 </div>
                 <small class="emms__fade-in"><strong>¡Ya puedes conocer a los speakers de EMMS Ecommerce!</strong><br>Accede y descubre quiénes son y las temáticas de sus conferencias</small>
-                <a href="./ecommerce.php" class="emms__cta emms__fade-in">CONOCE LA AGENDA</a>
+                <a href="./ecommerce" class="emms__cta emms__fade-in">RESERVA TU LUGAR AHORA</a>
```
Comment (mcappato) [inline index.php:256]: @MatiasRoy-Rodriguez @ornella-688  me confirman cual es el content correcto aqui?

File: registrado.php [hunk 3]
```diff
@@ -26,13 +26,13 @@
...
             <div class="emms__header__logo">
-                <a href="./index.php"><img src="src/img/logos/logo-emms.png" alt="Emms 2023"></a>
+                <a href="/"><img src="src/img/logos/logo-emms.png" alt="Emms 2023"></a>
             </div>
             <a class="emms__header__nav--mb" id="btn-burger"></a>
             <nav class="emms__header__nav emms__header__nav--hidden" id="nav-mb">
                 <ul class="emms__header__nav__menu">
                     <li><a href="#" class="active">home</a></li>
-                    <li><a href="./ecommerce-registrado.php">e-commerce</a></li>
+                    <li><a href="/ecommerce-registrado">ecommerce</a></li>
```
[the remark this answers is not in the document]
Comment (MatiasRoy-Rodriguez) [reply registrado.php:35]: e-commerce

File: index.php (hunk 2 above)
Comment (mcappato) [reply index.php:256]: done

**Pull request merged**